test/integration/adopter_profile_form_test.rb

```diff
@@ -74,34 +76,47 @@ class AdopterProfileFormTest < ActionDispatch::IntegrationTest
   end
 
   test "Phone number with less than 8 digits is invalid" do
-    sign_in users(:adopter_with_profile)
-    put "/adopter_profile",
-      params: {adopter_profile:
+    user = create(:user, :adopter_with_profile)
+    sign_in user
```

Comment:
Not a huge deal - seems like these 2 lines are duplicated enough that they could be extracted into their own method